.lintstagedrc.json

```diff
@@ -1,3 +1,3 @@
 {
-  "*.{js,jsx,ts,tsx}": ["prettier --write", "eslint --fix"]
+  "./**/*.{js,jsx,ts,tsx}": ["prettier --write", "eslint --fix"]
```

Comment:
@gpuente @acaldas I was having some trouble with the way lint-staged was configured. With the existing glob pattern, it was linting _everything_ not just what was staged. This meant I couldn't commit any code if I had unstaged changes with errors. 

I changed this because linting only the staged changes is the intended behavior.

Seems like this is a problem people run into quite often https://github.com/lint-staged/lint-staged/issues/869